TypeCobol/Compiler/CodeElements/Expressions/Expression.cs

```diff
@@ -66,6 +91,18 @@ public class LogicalOperation : ConditionalExpression
 
 		public ConditionalExpression RightOperand  { get; private set; }
 
+        protected override bool CheckEquivalence(Expression other)
+        {
+            Debug.Assert(other is LogicalOperation);
+            var otherLogicalOperation = (LogicalOperation)other;
+            return Operator.Value == otherLogicalOperation.Operator.Value;
+        }
+
+        public override (Expression, Expression) GetOperands()
```

Comment:
So it seems that this method GetOperands is not used now, I am thinking if it can be helpfull?

Reply:
It is planned to be used in external analyzers as it greatly simplifies the code.